ubuntu_18.04-dev.dockerfile

```diff
@@ -97,6 +97,11 @@ ENV LD_LIBRARY_PATH $HOME/.local/lib:$LD_LIBRARY_PATH
 
 RUN cd $HOME && nuc_data_make
 
-RUN cd pyne/tests \
+# Install OpenMC API
+RUN cd $HOME/opt && git clone https://github.com/openmc-dev/openmc.git
+RUN cd $HOME/opt/openmc && git checkout develop
+RUN pip install .
+
+RUN cd $HOME/opt/pyne/tests \
```

Comment:
Is there a reason this gets installed in the middle of the PyNE install/testing process? Should it come before PyNE as it is a dependency?

Reply:
Thanks @kkiesling